wings/data_source/coinbase_pro_api_order_book_data_source.py

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env python
+
+import asyncio
+import aiohttp
+import logging
+import pandas as pd
+from typing import (
+    Any,
+    AsyncIterable,
+    Dict,
+    List,
+    Optional,
+    Set,
+)
+import re
+import time
+import ujson
+import websockets
+from websockets.exceptions import ConnectionClosed
+
+from wings.orderbook.coinbase_pro_order_book import CoinbaseProOrderBook
+from .order_book_tracker_data_source import OrderBookTrackerDataSource
+from wings.order_book_tracker_entry import OrderBookTrackerEntry, CoinbaseProOrderBookTrackerEntry
```

Comment:
Please use the new multi-line import style (https://docs.python.org/2.4/whatsnew/node10.html)

```
from wings.order_book_tracker_entry import (
  OrderBookTrackerEntry,
  CoinbaseProOrderBookTrackerEntry
)
```